### PR TITLE
Update the RTC Engine to 0.7.0

### PR DESCRIPTION
**Pull request by jerzywilczek**

### CODE REVIEW

File: lib/videoroom/room.ex [hunk 1]
```diff
@@ -24,11 +24,18 @@ defmodule Videoroom.Room do
       id: room_id
     ]
 
+    mock_ip = Application.fetch_env!(:membrane_videoroom_demo, :external_ip)
+    external_ip = if @mix_env == :prod, do: {0, 0, 0, 0}, else: mock_ip
```
Comment (varsill) [inline lib/videoroom/room.ex]: `@mix_env` is not an inbuilt tag - you need to define it that way: `@mix_env Mix.env()`, as it was done here: https://github.com/membraneframework/membrane_videoroom/blob/4fc8841510086d8e9f91b71f9262ec1572216bc8/lib/videoroom/room.ex#L17

File: assets/package-lock.json [hunk 2]
```diff
@@ -4,16 +4,18 @@
   "requires": true,
```
Comment (varsill) [inline assets/package-lock.json:4]: I believe there is an inconsistency with the files package.json files ;< Haven't you installed dependencies with `npm install`?  
image

**Review by varsill — CHANGES_REQUESTED** [review]

**Comment by varsill** [comment]
General comment - we should probably explicitly say something about setting up EXTERNAL_IP env before running the application, as the application won't work with Firefox if the env is not set.

File: assets/package-lock.json (hunk 2 above)
Comment (jerzywilczek) [reply assets/package-lock.json:4]: I have and it worked. I'll check this again, perhaps it breaks after deleting `deps` or sth

Comment (jerzywilczek) [reply assets/package-lock.json:4]: It does work for me. I'll need more info on this, we should talk tomorrow.

**Review by varsill — APPROVED** [review]
lgtm! 🥇

**Pull request merged**